pyscriptjs/src/components/base.ts

```diff
@@ -232,6 +244,12 @@ function createWidget(name: string, code: string, klass: string) {
 
             this.wrapper = document.createElement('slot');
             this.shadow.appendChild(this.wrapper);
+
+            if (this.hasAttribute('pys-namespace')) {
```

Comment:
Since this is an attribute on PyScript components, I think we should name it simply `namespace`, without the `pys-` prefix.

Reply:
Makes sense! My only thought was to prevent created unnecessary namespaces in case other tags have a `namespace` attribute already, though I don't know of any other framework that would do so.

I suppose if we did run into that, it could be fixed by only generating namespaces for Pyscript-specific tags with the namespace attribute?

I'll change `pys-namespace` to `namespace` everywhere.